scene/src/utils/dcl-utils.ts

```diff
@@ -0,0 +1,4 @@
+import { timers } from '@dcl-sdk/utils'
+
+export const dclSleep = async (delay: number): Promise<any> =>
```

Comment:
```suggestion
export const delay = async (delay: number): Promise<any> =>
```

can be part of the utils.ts, no need for adding the dcl on the front of the name function

Reply:
this name is because it's using `import { timers } from '@dcl-sdk/utils`
so we don't confuse it with a classic sleep with native setTimeout that could be used at some moment in other cases in the project, e.g. maybe tests

btw, this function is not really used, it's a leftover now, I can delete it and add it again if necessary